packages/frontend/src/pages/Playground.svelte

```diff
@@ -67,8 +70,14 @@ let sendEnabled = $derived.by(() => {
   return false;
 });
 
-let server = $derived(
-  $inferenceServers.find(is => conversation && is.models.map(mi => mi.id).includes(conversation?.modelId)),
+$effect(() => {
+  if (latest && isErrorMessage(latest)) {
+    errorMsg = latest.error;
+  }
+});
```

Comment:
hello, wondering why it's not also a derived state ?

something like
```
errorMsg = $derived(latest && isErrorMessage(latest) ? latest.error : undefined)
```

Reply:
because when calling the `submit` method, we put the potential error also inside this state

https://github.com/containers/podman-desktop-extension-ai-lab/blob/dafb8807898380133ae8086c6aab1e0549ab2c8d/packages/frontend/src/pages/Playground.svelte#L89-L91